### ISSUE TITLE
Is FontAwesome ISO Compliant?

### ISSUE THREAD

**Issue by phasjim** [FortAwesome/Font-Awesome#9455]
Hello! I've looked everywhere on the FA documentation, but I can't seem to find it so I just thought I would ask. I was wondering if Font Awesome is ISO Compliant?

http://smallbusiness.chron.com/iso-compliant-61481.html

Let me know! Thanks 😄

**Comment by Crissov** [comment]
No, Font Awesome and Fort Awesome are not certified according to ISO 9000 or 9001, at least as far as I can see, and that’s quite normal for open source software, although a proper Github repository already contains much of the documentation required by these Quality Assurance standards.

The files released by FA should comply to other ISO standards, though (e.g. OTF is ISO/IEC 14496-22).

**Comment by sensibleworld** [comment]
Doing a little management in Github and (batch) closing out potentially stale issues. If the above still needs attention, please let us know and we’ll re-open!
